apps/sms/test/marionette/lib/views/conversation/message_accessors.js

```diff
@@ -39,16 +39,20 @@ MessageAccessor.prototype = {
         content: messageNode.findElement(SELECTORS.content).text(),
         attachments: attachments.map(function(attachment) {
           return {
+            screenshot: function() {
+              return this.client.screenshot({ element: attachment });
+            }.bind(this),
```

Comment:
I didn't want to take the screenshot for every messages so I used a function instead -- this kind of defeats the goal that we have a snapshot of the state though, so tell me what you think.